trunion/validators.py

```diff
@@ -45,13 +46,17 @@ def valid_receipt(request):
     if receipt['iss'] not in request.registry.settings['trunion.permitted_issuers']:
         raise HTTPConflict("Bad issuer: \"%s\"" % receipt['iss'])
```

Comment:
sure but those won't have the actual values such as the timestamps we are comparing so we'd have no way to debug clock skew without pushing logging changes to prod. I wouldn't advise putting actual values in the exception messages (since those might get exposed) but adding logging would be great.